src/domain/event/__tests__/EventPage.test.tsx

```diff
@@ -636,4 +636,56 @@ describe('refetch of event works correctly', () => {
       ).toBeInTheDocument();
     });
   });
+
+  it('renders a link which leads to all events of the organisation', async () => {
+    renderComponent();
+    await waitForRequestsToComplete();
+    expect(
+      screen.queryByText('Näytä järjestäjän tapahtumat')
+    ).toBeInTheDocument();
+  });
```

Comment:
Could we check also
```
 data.contactPersonName,
      data.contactPersonEmail,
      data.contactPersonPhoneNumber,
```
are present?

Reply:
nitpick but use `queryBy` only with `not.toBeInTheDocument` and findBy otherwise. It could be:

```
[
      data.contactPersonName,
      data.contactPersonEmail,
      data.contactPersonPhoneNumber,
    ].forEach(async (text) => await screen.findByText(text);
```
or could this work?
```
await promise.all([
      data.contactPersonName,
      data.contactPersonEmail,
      data.contactPersonPhoneNumber,
    ].map(screen.findByText);
```